_config/routes.yml

```diff
@@ -1,6 +1,7 @@
 ---
 Name: testsessionroutes
 ---
-SilverStripe\Control\Director:
-  rules:
-    dev/testsession: SilverStripe\TestSession\TestSessionController
+SilverStripe\Dev\DevelopmentAdmin:
+  registered_controllers:
+    testsession:
+      controller: SilverStripe\TestSession\TestSessionController
```

Comment:
Can you rename the key here to something like testsession to avoid conflicts?

Reply:
Correct me if I'm wrong but I think the key is set on `line 6` as `testsession`? And `controller:` is meant to be the right key as we're specifying which controller to use?

I also tried changing `controller` to `testsession` and that throws an error..

Reply:
Sorry you’re right, I skim read